tests/sources/test_jenkins_job_builder.py

```diff
@@ -0,0 +1,132 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+# pylint: disable=no-member
+import os
+import shutil
+from unittest import TestCase
+from unittest.mock import Mock, patch
+
+from cibyl.models.ci.system import System
+from cibyl.sources.jenkins_job_builder import JenkinsJobBuilder
+
+
+def remove_fake_files():
+    """Remove the fake xml files."""
+    shutil.rmtree("out_jjb_test")
+
+
+# pylint: disable=unspecified-encoding
```

Comment:
This one is fixed by adding 'open(..., encoding='utf-8')' I believe. There are other parts in the code with this.

Reply:
You're right, I'll add it